src/builder.ts

```diff
@@ -17,9 +17,114 @@ import assert = require('assert');
 import { Server } from 'http';
 import logo from './initializers/logo';
 
+export class OrkaBuilder {
```

Comment:
💣

Reply:
Does this need to be exported?

Reply:
It complains here: 
https://github.com/Workable/orka/blob/master/src/index.ts#L4
and here:
https://github.com/Workable/orka/blob/master/src/orka.ts#L6

if i don't export it